Service/CredentialsRevokerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Service;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+
+interface CredentialsRevokerInterface
```

Comment:
@toniperic Since we know this is going to be apart of our public API, please add comments in the class header that describe the interface's purpose (same goes for the method itself).